### PR TITLE
Express-first-API

### PR DESCRIPTION
**Pull request by LillanEriksson**
Wish I had more time to work on this project. I only had the possibility to do the minimum requirements this week due to stressful times at work.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,33 +1,71 @@
...
+	const showWin = req.query.win;
+	let nominationsFromYear = goldenGlobesData.filter(
+		(item) => item.year_award === +year
+	);
+
+	if (showWin) {
+		nominationsFromYear = nominationsFromYear.filter((item) => item.win);
+	}
+	res.json(nominationsFromYear);
+});
```
Comment (HIPPIEKICK) [inline server.js:46]: Endpoints should be named after what they return and not how they are filtered, so these could be query params under your `/nominations` endpoint:
`nominations?year=2010&win=true`

File: server.js [hunk 2]
```diff
@@ -1,33 +1,71 @@
...
+
+//single item
+
+app.get('/year/:year/category/best_drama/?win=true', (req, res) => {
+	const year = req.params.year;
+	const showWin = req.query.win;
+
+	let bestDrama = goldenGlobesData.filter(
+		(item) => item.category === 'Best Motion Picture - Drama'
```
Comment (HIPPIEKICK) [inline server.js:55]: Would be nice if this was also a query param under the `nominations` endpoint 🤩 
`nominations?year=2010&win=true?category=drama`

Comment (HIPPIEKICK) [reply server.js:55]: Oh, this endpoint doesn't seem to be working 🤔 `?win=true` shouldn't be in the naming of the endpoint, but rather just a query param - right?

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Fix the broken endpoint and you're good to go 😇

File: server.js (hunk 2 above)
Comment (LillanEriksson) [reply server.js]: Changed the endpoint and it's working now

Comment (HIPPIEKICK) [reply server.js]: Looks like you removed it, or did you not commit your latest changes maybe. The requirements say to have one endpoint that only shows a single result

Comment (LillanEriksson) [reply server.js]: Fixed it now, changed dataset to have an ID to search for to show a single item

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**